primeiro

thaislisatchok opened the pull request.

thaislisatchok committed:
primeiro

MatheusPinto commented:
Coloque o README.md dentro da pasta "Thaís". Mas mantenha o README.md na raiz sem texto algum.

MatheusPinto reviewed (changes requested):
Coloque o README.md dentro da pasta "Thaís". Mas mantenha o README.md na raiz sem texto algum.

thaislisatchok committed:
Alterações no README.md e adição de "pastas"

Como havia solicitado colocamos o README.md na pasta "Thaís_Guido" e mantivemos o README.md na raiz sem texto algum. Além disso, criamos pastas individuais para cada etapa do projeto, facilitando a navegação e a localização dos arquivos relacionados a cada etapa.

-
